backend/pkg/rpconnect/docs/sync.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+cp ../../../../go/benthos/internal/docs/* .
+rm json_schema.go benchmark_test.go bloblang_markdown.go
+sed -i 's/github.com\/benthosdev\/benthos\/v4\/internal\/docs/dullploy\/blobstudio\/internal\/docs/g' ./*.go
```

Comment:
Let's delete this file as well. The reason this exists is because we essentially have a fork of the `benthos/internal/docs` so that we can implement our tree builder, and this script lets us quickly bring in bug fixes. However, I think in reality we will go with one of these two options going forwards:

1. Reduce this package and maintain it separately
2. Remove this and we add the tree builder to benthos

Option 2 is definitely preferred, but it would also be a public API on Benthos which would need to adhere to backwards compatibility. Therefore, I need a bit of time to figure out what this functionality would look like, and so option 1 will need to be enough for now.